go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/json-iterator/go v1.1.6
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/stretchr/testify v1.3.0 // indirect
```

Comment:
I don't think that's ok though? What uses it?

Reply:
I don't know, I looked a while back, it was some dependency I think.
Every time I run go mod tidy that gets added.

Reply:
`go mod why github.com/stretchr/testify/...`
```
# github.com/stretchr/testify
(main module does not need package github.com/stretchr/testify)

# github.com/stretchr/testify/assert
sigs.k8s.io/structured-merge-diff/v3/fieldpath
github.com/json-iterator/go
github.com/json-iterator/go.test
github.com/stretchr/testify/require
github.com/stretchr/testify/assert

# github.com/stretchr/testify/http
(main module does not need package github.com/stretchr/testify/http)

# github.com/stretchr/testify/mock
(main module does not need package github.com/stretchr/testify/mock)

# github.com/stretchr/testify/require
sigs.k8s.io/structured-merge-diff/v3/fieldpath
github.com/json-iterator/go
github.com/json-iterator/go.test
github.com/stretchr/testify/require

# github.com/stretchr/testify/suite
(main module does not need package github.com/stretchr/testify/suite)
```

Reply:
go.mod includes all dependencies, even test ones